Level Solutions/10981.0.txt

```diff
@@ -0,0 +1,31 @@
+@0
+
+#Adding whatever number follows to the acc and moving the acc down

```

Comment:
Comments should explain "why", not just "what". I.e. explain that the purpose of adding `ACC` to itself is to double the value.